src/web/utils/textHelpers.ts

```diff
@@ -33,3 +33,11 @@ export const separateStringsList = (strings: string): string[] => {
 export const deduplicateStrings = (strings: string[]) => {
   return strings.filter((val, index, arr) => arr.indexOf(val) === index);
 };
+
+export const sortStrings = (strings: string[]) => {
```

Comment:
perhaps worth calling this something like `sortStringsAlphabetically` to avoid confusion with [natural sort](https://en.wikipedia.org/wiki/Natural_sort_order)

Reply:
renamed